engine/systemdnspawn/systemdnspawn.go

```diff
@@ -47,6 +47,25 @@ func (e Engine) Run(command string, args []string, environment types.Environment
 		}
 		nspawncmd = append(nspawncmd, "--chdir", workingDir)
 	}
+	if systemdVersion >= 230 {
+		machineIdFile := path.Join(chroot, "/etc/machine-id")
+		_, err := os.Stat(machineIdFile)
```

Comment:
In this case there can only be one acbuild acting at a time on this build due to the lock it grabs when a command starts.